From 84b8b5b46773f9b686d57f28092824b86bffed9d Mon Sep 17 00:00:00 2001
From: Mikael Magnusson <mikachu@gmail.com>
Date: Sun, 11 Mar 2012 01:41:55 +0000
Subject: Fixup for 550b2f76401c2 which broke bold fonts

The commit removed the line
AC_CHECK_FUNCS(FT_Get_Next_Char FT_Get_BDF_Property FT_Get_PS_Font_Info FT_Has_PS_Glyph_Names FT_GlyphSlot_Embolden)
but failed to remove the #if lines that this commit removes, resulting
in that code never being executed.

Fixes: https://bugs.freedesktop.org/attachment.cgi?id=58280

Reviewed-by: Jeremy Huddleston <jeremyhu@apple.com>
---
diff --git a/src/xftglyphs.c b/src/xftglyphs.c
index 4a3b9d6..2f3dc5a 100644
--- a/src/xftglyphs.c
+++ b/src/xftglyphs.c
@@ -24,9 +24,7 @@
 #include <freetype/ftoutln.h>
 #include <freetype/ftlcdfil.h>
 
-#if HAVE_FT_GLYPHSLOT_EMBOLDEN
 #include <freetype/ftsynth.h>
-#endif
 
 /*
  * Validate the memory info for a font
@@ -434,12 +432,10 @@ XftFontLoadGlyphs (Display	    *dpy,
 
 	glyphslot = face->glyph;
 
-#if HAVE_FT_GLYPHSLOT_EMBOLDEN
 	/*
 	 * Embolden if required
 	 */
 	if (font->info.embolden) FT_GlyphSlot_Embolden(glyphslot);
-#endif
 
 	/*
 	 * Compute glyph metrics from FreeType information
--
cgit v0.9.0.2-2-gbebe
